test/utils.d

```diff
@@ -22,6 +22,27 @@ string trelloTestHookURL;
 string payloadDir = "./data/payloads";
 string hookDir = "./data/hooks";
 
+/// Tries to find a free port
+ushort getFreePort()
+{
+    version(Windows)
```

Comment:
The below solution should also work on Windows. See: https://stackoverflow.com/a/9895416/1808755